config/settings.d/dns_dnscmd.yml.example

```diff
@@ -0,0 +1,8 @@
+---
+# Can be true, false
+:enabled: true
+
+# use this setting if you are managing a dns server which is not localhost though this proxy
```

Comment:
This is identical to the comment in nsupdate - can we make this easier to distinguish when I would use either?